Removes -L${libdir} in OpenEXR.pc.in
Will not send upstream.

--- OpenEXR.pc.in	2016-11-17 11:34:35.415294387 +0000
+++ OpenEXR.pc.in	2016-11-17 11:35:14.147488887 +0000
@@ -8,7 +8,7 @@
 Description: OpenEXR image library
 Version: @OPENEXR_VERSION@
 
-Libs: -L${libdir} -lIlmImf
+Libs: -lIlmImf
 Cflags: -I${OpenEXR_includedir}
 Requires: IlmBase
 Libs.private: -lz
